features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -3830,6 +3830,25 @@
         </OverwriteDispatch>
     </TenantContextsToRewrite>
 
+    <OrgContextsToRewriteInTenantPerspective>
+        <WebApp>
```

Comment:
Hope we don't have a usecase of adding new contexts through deployment.toml as of now.

Reply:
Yes we dont. we can introduce the configurable facility afterwards.